PowerEditor/src/Parameters.h

```diff
@@ -1939,6 +1942,11 @@ class NppParameters final
 
 	LanguageNameInfo getLangNameInfoFromNameID(const std::wstring& langNameID);
 
+	void setNbTabCompactLabelLen(UINT nb) {
+		_nppGUI._tabCompactLabelLen = nb;
+	};

```

Comment:
@donho 
it is possible to update https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#general to remove redundant semicolon, some static analyzer are detecting it as warning,

```c++
class MyClass
{
public:
    void method1();
    int method2() {
        return _x; // only one line code can be placed in .h as method definition
    }; // <<< remove this redundant semicolon

private:
    int _x;
}
```

Reply:
@ozone10 
It's correct & logic syntax IMO.

> some static analyzer are detecting it as warning

Which one?

Reply:
@donho 
results of adding flag -Wextra-semi
https://github.com/ozone10/notepad-plus-plus/actions/runs/19009777568/job/54288998446
28342 warnings